src/feature/DeckBuilder/index.tsx

```
@@ -0,0 +1,705 @@
+      // Species filter
+      const matchesSpecies =
+        selectedSpecies.length === 0 ||
+        (catalog.species && catalog.species.some(s => selectedSpecies.includes(s)));
+
+      // Originality filter
+      const matchesOriginality =
+        selectedOriginalities.length === 0 || selectedOriginalities.includes(catalog.originality);
+
+      // Version filter
+      const matchesVersion =
+        selectedVersions.length === 0 || selectedVersions.includes(catalog.info.version);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)